### PR TITLE
An attempt at adding more prisoner crime options

### PR DESCRIPTION
**Pull request by Aether217**
## About The Pull Request

This PR adds more prisoner crimes to the bunch. They are crimes from the nova corporate regulations and contraband table, meaning that they should blend in naturally with the game.

## How This Contributes To The Nova Sector Roleplay Experience

Prisoners having more details about why they are inside is always good, it's more material for the roles that interact with prisoners.

## Proof of Testing
<details>
<summary>Screenshots/Videos</summary>
  
</details>

## Changelog
:cl:
add: 4 new crime options
add: Added more things
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
sound: added/modified/removed audio or sound effects
image: added/modified/removed some icons or images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by FlufflesTheDog** [comment]
You could _probably_ get away with moving these upstream, or if it doesn't fit there you'll need to modularize this.

**Comment by Aether217** [comment]
> You could _probably_ get away with moving these upstream, or if it doesn't fit there you'll need to modularize this.

I'll give a look to modularizing this later but I have little idea of what I'm doing, I am but an ape with access to internet.

**Comment by FlufflesTheDog** [comment]
Here's a guide for it, and feel free to ask in the coding channel on discord if you want.
https://github.com/NovaSector/NovaSector/blob/master/modular_nova/readme.md

Because these can be added without modifying existing code, best option will probably be to mirror the file in the modular folder - eg `modular_nova/master_files/code/modules/client/preferences/prisoner_crime.dm` and put the new crimes there.

File: code/modules/client/preferences/prisoner_crime.dm [hunk 1]
```diff
@@ -148,3 +148,23 @@ GLOBAL_LIST_INIT(prisoner_crimes, init_prisoner_crimes())
...
+	tattoos = 2
+
+/datum/prisoner_crime/boom_man
+	name = "Possession of unauthorised explosives."
+	desc = "Being in possession of restricted explosives."
+	tattoos = 2
+
+/datum/prisoner_crime/self_destructive
+	name = "Self-Destructive Tendencies"
+	desc = "The complete disregard for personnal safety shown by repeatedly putting oneself in death or near death situations." //this is mainly for protective custody prisoners
```
[the remark this answers is not in the document]
Comment (Aether217) [reply code/modules/client/preferences/prisoner_crime.dm:169]: I'll do it here modular when I have my next bit of free time, I want to eventually add nicer things to Nova so I want to get used to small PRs first.

**Pull request closed without merging**